indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,110 @@
         }
     }
 }
+
+.display-contribution-list {
+    @extend %font-family-modern-body;
+
+    .contribution-row {
+        @include border-left($width: 3px);
+        padding-left: 10px;
+        margin-bottom: 1em;
+
+        &:hover {
+            border-color: $blue;
+        }
+        .row-title {
+            @include border-bottom();
+            font-size: 14pt;
+        }
+        .submitter {
+            color : $light-black;
+            margin-top: 7px;
+        }
+        .label {
+            border: 0;
+            margin-top: 7px;
+            display: inline-block;
+            padding: 5px 7px;
+
+            &.review-role {
+                @include border-all($gray);
+                margin-right: 10px;
+                background-color: white;
+                color: $light-black;
+                i {
+                    color: $blue;
+                }
+            }
+
+            &.submitted {
```

Comment:
Isn't there a way to do the same using the `.semantic-background` class (which doesn't exist currently, but it could be easily implemented similar to `.semantic-text`